sessions/js-array-methods-2/demo-start/js/index.js

```diff
@@ -1,54 +1,86 @@
-
-const studentNames = ["Lucille", "Gilbert", "Jennie", "Lydia", "Agnes"];
-
-const students = [
-  { name: "Lucille", age: 30, points: 26, happiness: 5 },
-  { name: "Gilbert", age: 45, points: 0, happiness: 1 },
-  { name: "Jennie", age: 47, points: 39, happiness: 5 },
-  { name: "Lydia", age: 29, points: 14, happiness: 2 },
-  { name: "Agnes", age: 34, points: 39, happiness: 4 },
+const simpsons = [
+  {
+    id: "13e25ab",
+    name: "Homer",
+    img: "https://cdn.glitch.com/3c3ffadc-3406-4440-bb95-d40ec8fcde72%2FHomerSimpson.png?1497567511939",
+    age: 45,
+    income: 3500,
+  },
+  {
+    id: "13dfgd5ab",
+    name: "Marge",
+    img: "https://cdn.glitch.com/3c3ffadc-3406-4440-bb95-d40ec8fcde72%2FMargeSimpson.png?1497567512205",
+    age: 43,
+    income: 0,
+  },
+  {
+    id: "gwd4gd5ab",
+    name: "Ape",
+    img: "https://cdn.glitch.com/3c3ffadc-3406-4440-bb95-d40ec8fcde72%2FAbrahamSimpson.png?1497567511593",
+    age: 83,
+    income: 500,
+  },
+  {
+    id: "dr325ab",
+    name: "Bart",
+    img: "https://cdn.glitch.com/3c3ffadc-3406-4440-bb95-d40ec8fcde72%2FBartSimpson.png?1497567511638",
+    age: 15,
+    income: 50,
+  },
+  {
+    id: "13erf25ab",
+    name: "Lisa",
+    img: "https://cdn.glitch.com/3c3ffadc-3406-4440-bb95-d40ec8fcde72%2FLisaSimpson.png?1497567512083",
+    age: 12,
+    income: 35,
+  },
+];
+const numbers = [512, 77, 333, 49, 81, 4, -12, 3.14, 1000, 31, 99];
+const strings = [
+  "HTML",
+  "React",
+  "CSS",
+  "Next.js",
+  "MongoDB",
+  "styled components",
+  "mongoose",
+  "next-auth",
+  "Visual Studio Code",
 ];
 
-handleList(students, "Original Students");
-
-/** includes */
-
-const studentNamesIncludesHarry = null;
-
-handleSingleValue(studentNamesIncludesHarry, "studentNamesIncludesHarry");
-
-/** find and findIndex */
-
-const personOverThirty = null;
-
-// handleObject(personOverThirty, 'personOverThirty');
-
-const indexOfStudentWith39Points = null;
-
-// handleSingleValue(indexOfStudentWith39Points, 'indexOfStudentWith39Points');
-
-/** sort */
-
-const sortedByAge = null;
-
-// handleList(sortedByAge, 'sortedByAge');
-
-const sortedByName = null;
-
-// handleList(sortedByName, 'sortedByName');
-
-/** slice */
-
-// handleList(students, 'Not Original Anymore');
-// handleList(sortedByAge, 'Not Sorted By Age Anymore');
-
-/** some and every */
-
-const anyStudentHasZeroPoints = null;
-
-// handleSingleValue(anyStudentHasZeroPoints, 'anyStudentHasZeroPoints');
-
-const everyStudentIsOlderThanThirty = null;
+const characterList = document.querySelector("[data-js='characterList']");
+const numberList = document.querySelector("[data-js='numberList']");
+const numberOutput = document.querySelector("[data-js='number-output']");
+const stringList = document.querySelector("[data-js='stringList']");
+const stringOutput = document.querySelector("[data-js='string-output']");
+const findOutput = document.querySelector("[data-js='find-output']");
+
+
+numbers.map((number) => {
```

Comment:
Since we are just looping but not assigning to anything new, you can use `forEach` instead ^^